packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -22,14 +22,21 @@ import { getRequestedVersion } from "./versionUtils.js";
  * Generate configuration combinations for a particular compat version
  * NOTE: Please update this packages README.md if the default versions and config combination changes
  */
```

Comment:
Looks like this comment was moved to the new interface from the old one. Is that correct?

Reply:
Ah good catch, thanks!